Dockerfile

```diff
@@ -46,18 +46,20 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
 ARG package=.
 ARG ARCH
 ARG ldflags
+ARG outbin="manager"
+ENV execbin=$outbin
```

Comment:
I believe this is because `ENTRYPOINT` does not support `ARG` values, however it does support using environment variables set using `ENV`.

Ref: https://docs.docker.com/engine/reference/builder/#environment-replacement

Reply:
Ah interesting